DependenSee/ReferenceDiscoveryService.cs

```diff
@@ -130,6 +139,11 @@ private void Discover(string folder, DiscoveryResult result)
             }
         }
         var directories = Directory.EnumerateDirectories(folder);
+        if (_shouldIncludeTransitiveProjectDependencies)
```

Comment:
If the option is on, find the directory for all of the projects this project depends on, and add all of them that aren't already in the list to the list of directories to run discovery on.  This lets one chase a dependency hierarchy for a single project all the way to the root through lots of directories and build a much more meaningful graph than one that just has it's projects and the single projects it depends on (when you run it focused on a single project directory in a solution for example).